lib/ansible/modules/packaging/os/pulp_repo.py

```diff
@@ -48,27 +48,34 @@
     type: bool
     default: 'no'
     version_added: "2.8"
-  ca_cert:
+  feed_ca_cert:
     description:
       - CA certificate string used to validate the feed source SSL certificate.
         This can be the file content or the path to the file.
+        The ca_cert alias will be removed in Ansible 2.14.
     type: str
-    aliases: [ importer_ssl_ca_cert ]
-  client_cert:
+    aliases: [ importer_ssl_ca_cert, ca_cert ]
+  feed_client_cert:
+    version_added: "2.10"
     description:
       - Certificate used as the client certificate when synchronizing the
         repository. This is used to communicate authentication information to
         the feed source. The value to this option must be the full path to the
         certificate. The specified file may be the certificate itself or a
         single file containing both the certificate and private key. This can be
         the file content or the path to the file.
+        If not specified the default value will come from client_cert. Which will
```

Comment:
I would add this to a new paragraph, so it will be easier to read (and harder to miss).
```suggestion
      - If not specified the default value will come from client_cert. Which will
```